### ISSUE TITLE
[BUG] App crashes sometines when opening share by link

### ISSUE THREAD

**Issue by jesmrec** [owncloud/android#2882]
By executing automatic tests with Appium, the following crash happens sometimes when creating public link. It seems to happen after matching the name of the public link.

```
2020-05-19 17:27:09.084 7719-7826/io.appium.uiautomator2.server I/appium: Find element command using 'id' with selector 'com.owncloud.android:id/shareFileName'.
2020-05-19 17:27:09.184 7719-7828/io.appium.uiautomator2.server I/appium: Find element command using 'id' with selector 'com.owncloud.android:id/addPublicLinkButton'.
2020-05-19 17:27:09.272 7719-8004/io.appium.uiautomator2.server I/appium: Find element command using 'id' with selector 'com.owncloud.android:id/shareViaLinkNameValue'.
2020-05-19 17:27:09.295 12142-12142/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.owncloud.android, PID: 12142
    kotlin.KotlinNullPointerException
        at com.owncloud.android.presentation.ui.sharing.fragments.PublicShareDialogFragment.updateInputFormAccordingToServerCapabilities(PublicShareDialogFragment.kt:640)
        at com.owncloud.android.presentation.ui.sharing.fragments.PublicShareDialogFragment.updateCapabilities(PublicShareDialogFragment.kt:624)
        at com.owncloud.android.presentation.ui.sharing.fragments.PublicShareDialogFragment.access$updateCapabilities(PublicShareDialogFragment.kt:63)
        at com.owncloud.android.presentation.ui.sharing.fragments.PublicShareDialogFragment$observeCapabilities$1.invoke(PublicShareDialogFragment.kt:447)
        at com.owncloud.android.presentation.ui.sharing.fragments.PublicShareDialogFragment$observeCapabilities$1.invoke(PublicShareDialogFragment.kt:63)
        at com.owncloud.android.domain.utils.Event$EventObserver.onChanged(Event.kt:55)
        at com.owncloud.android.domain.utils.Event$EventObserver.onChanged(Event.kt:52)
        at androidx.lifecycle.LiveData.considerNotify(LiveData.java:131)
        at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:149)
        at androidx.lifecycle.LiveData.setValue(LiveData.java:307)
        at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
        at androidx.lifecycle.LiveData$1.run(LiveData.java:91)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by jesmrec** [comment]
I could also reproduce when creating a public link after a fresh install, and no internet connection

**Comment by jesmrec** [comment]
Fixed, to release in 2.15
